airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/SecretsRepositoryReader.java

```diff
@@ -75,6 +77,7 @@ public DestinationConnection getDestinationConnectionWithSecrets(final UUID dest
 
     hydrateValuesIfKeyPresent(sourceKey, dump);
     hydrateValuesIfKeyPresent(destinationKey, dump);
+    hydrateStagingConfig(dump);
```

Comment:
Why are we exporting staging configs? I know it's not intended for use in OSS, but let's say we release a paid version of Airbyte self-hosted where this feature is enabled. Then we probably want to permission what can be dumped individually, rather than allow the user to dump the whole instnace. 

@jrhizor does the platform already have a stance on this? My vague feeling is we shouldn't dump at all here but would love feedback from you

How do we want to handle secrets which are really only meant for use in Cloud/paid?